crates/isograph_config/src/compilation_options.rs

```diff
@@ -203,6 +209,10 @@ impl Default for ConfigFileOptionalValidationLevel {
         Self::Error
     }
 }
+pub fn generate_json_schema_config() {
```

Comment:
Can you move this to `main.rs`? Seems weird to have side effectful functions here